handler.go

```diff
@@ -115,6 +118,101 @@ type TorrentManager struct {
 	Updates time.Duration
 
 	hotCache *lru.Cache
+
+	// For manage torrents Seeding by SeedingLocal(), true/false means seeding/pause
+	localSeedLock  sync.RWMutex
+	localSeedFiles map[string]bool
+}
+
+// can only call by fs.go: 'SeedingLocal()'
+func (tm *TorrentManager) addLocalSeedFile(ih string) bool {
+	if !common.IsHexAddress(ih) {
+		return false
+	}
+	ih = strings.TrimPrefix(strings.ToLower(ih), common.Prefix)
+
+	tm.localSeedLock.Lock()
+	defer tm.localSeedLock.Unlock()
+
```

Comment:
Move ```tm.localSeedLock.Lock()
	defer tm.localSeedLock.Unlock()``` 2 lines down below ```GoodFiles``` validation for ``` GoodFiles``` is static

Reply:
Got